src/server/routes/api/v1/parking-lot.ts

```diff
@@ -0,0 +1,113 @@
+import httpResponse from '../../../utils/http-response';
+import type {
+  FastifyError,
+  FastifyInstance,
+  FastifyRegisterOptions,
+  FastifyReply,
+  FastifyRequest,
+} from 'fastify';
+import { UnexistentParkingLotID } from '../../../error/parking-lot.service';
+
+export default function (
+  fastify: FastifyInstance,
+  _: FastifyRegisterOptions<unknown>,
+  done: (err?: FastifyError) => void,
+): void {
+  fastify.post(
+    '/',
+    async (
+      request: FastifyRequest<{
+        Body: {
+          name: string;
+        };
+      }>,
+      reply: FastifyReply,
+    ) => {
+      try {
+        const parkingLot = await fastify.services.parkingLot.create(
+          request.body,
+        );
+
+        reply.status(201);
+        return httpResponse.created(reply, parkingLot);
+      } catch (error) {
+        return httpResponse.internalServerError(reply, error);
+      }
+    },
+  );
+
+  fastify.post(
+    '/:id',
+    async (
```

Comment:
We are still missing the schema here. Please follow:
https://github.com/whizzes/thruway-parking/blob/9050cba18896b6bb52ecdc3c84bf365247f58b74/src/server/routes/api/v1/users/index.ts#L21